src/main/java/seedu/address/logic/commands/AddAltCommand.java

```diff
@@ -47,16 +48,16 @@ public class AddAltCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_TELEGRAM + "johndoe_telegram"
             + PREFIX_SECONDARY_EMAIL + "johndoe2@example.com";
-    public static final String MESSAGE_ADDALT_SUCCESS = "Added alternative contact to Person: %1$s";
-    public static final String MESSAGE_NO_ADDALT = "At least one field to edit must be provided.";
+    public static final String MESSAGE_ADDALT_SUCCESS = "Added alternative contact information to Person: %1$s";
     public static final String MESSAGE_ADDALT_DUPLICATE_EMAIL = "Secondary email is same as primary email "
             + "for Person: %1$s";
-    public static final String MESSAGE_ADDALT_FAILURE = "There is existing alternative contact to Person: %1$s. "
-            + "To change the corresponding field, use the edit command.";
+    public static final String MESSAGE_ADDALT_FAILURE = "There is existing %1$sto Person: %2$s. "
```

Comment:
Wondering if this could be phrased a bit more clearly, `"There is existing Telegram Birthday to Person XXX"` is a little bit of an odd phrasing.

Perhaps something like `"Person XXX already has the following fields (Telegram, ...). \nTo change the ..."`
Then for the implementation below, instead of appending to a string, you can consider adding to an `ArrayList<String> existingAlternativeContactFields` and using [`String#join`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-) to form the string you need. This way you won't need to worry about trailing commas/spaces.

Reply:
Also just an FYI, not a big deal. [Explicit indexing](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#:~:text=in%20three%20ways%3A-,Explicit%20indexing,-is%20used%20when) for format strings is probably only necessary if you are going to reuse arguments or reorder them. In this case it should be unnecessary as the arguments are in order and not reused.

Reply:
thank you for pointing this out! was really unsure of how to make the phrasing better and whether there are much better implementations to do so. thanks for the guidance! fixed in ed11324!